framework/src/main/java/org/checkerframework/framework/util/element/ElementAnnotationUtil.java

```diff
@@ -499,11 +531,11 @@ private static AnnotatedTypeMirror getLocationTypeAWT(
         if (!location.isEmpty()
                 && location.get(0).tag.equals(TypeAnnotationPosition.TypePathEntryKind.WILDCARD)) {
             if (AnnotatedTypes.hasExplicitExtendsBound(type)) {
-                return getTypeAtLocation(type.getExtendsBound(), tail(location));
+                return getTypeAtLocation(type.getExtendsBound(), tail(location), null, false);
```

Comment:
How about adding an overload of `getTypeAtLocation` that passes `null, false`.
Then most invocations won't need to be changed and the few that need change can be understood and documented.